turbinia/e2e/setup-gce.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+# This script will
+# - setup a GCE Ubuntu 18.04 LTS base image for Turbinia e2e testing.
+# - execute Terraform to setup the Turbinia environment.
+# Make sure the service account your GCE instance is running under has full API scope
+# access and is project owner for Terraform to function correctly.
+
+install_packages() {
+  apt-get -y update
+  apt-get -y install python-pip python-virtualenv unzip
+}
+
+echo "Initiate gcloud configuration"
+
+if [ $# -ne  1 ]
+then
+  echo "Not enough arguments supplied, please provide project."
+  echo "$0 [PROJECT]"
+  exit 1
+fi
+
+PROJECT="$1"
+
+gcloud --project=$PROJECT info
+
+# Try to install the packages. GCE instances do their own apt-get updating at
+# first boot so we have to wait on that to install our packages.
+max_retry=100
+for try in $(seq 1 ${max_retry}); do
+  [[ ${try} -gt 1 ]] && sleep 10
+  install_packages && exit_code=0 && break || exit_code=$?
```

Comment:
Is `exit_code` actually used anywhere?

Reply:
It's used on the same line, it's a bit of a one liner but the exit code is 0 if install_packages succeeds and the loop breaks, or it is not 0 and the loop continues.